mlx-rs/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "mlx-rs"
+version = "0.14.0"
+authors = [
+  "Minghua Wu <michael.wu1107@gmail.com>",
+  "David Chavez <david@dcvz.io>",
+]
+edition = "2021"
```

Comment:
not a big deal but these can be grabbed from workspace Cargo.toml

Reply:
Didn't see this before I hit the squash and merge button lol. We could do this later then.